metis-normalization/src/main/java/eu/europeana/normalization/dates/extraction/dateextractors/NumericPartsDateExtractor.java

```diff
@@ -37,39 +39,44 @@ public class NumericPartsDateExtractor implements DateExtractor {
   private static final String UNKNOWN_CHARACTERS_REGEX = "[XU?-]";
 
   @Override
-  public DateNormalizationResult extract(String inputValue) {
-    return extract(inputValue, NumericPartsPattern.NUMERIC_SET);
+  public DateNormalizationResult extract(String inputValue, DateQualification requestedDateQualification,
+      boolean flexibleDateBuild) throws DateExtractionException {
+    return extract(inputValue, requestedDateQualification, NumericPartsPattern.NUMERIC_SET, flexibleDateBuild);
   }
 
   /**
    * Validates and returns a date normalization result of provided input value.
    *
    * @param inputValue the input value
    * @param numericPatternValues the patterns to check against
+   * @param allowSwitchMonthDay allow switching month and day values if month and day original values are not valid
    * @return the date normalization result
    */
-  public DateNormalizationResult extract(String inputValue, Set<NumericPartsPattern> numericPatternValues) {
+  protected DateNormalizationResult extract(String inputValue, DateQualification requestedDateQualification,
+      Set<NumericPartsPattern> numericPatternValues,
+      boolean allowSwitchMonthDay) throws DateExtractionException {
```

Comment:
Similarly like before I coundn't think of something better when writing this.  
That is because it is just required for on call in the builder and not a bunch of things.  
In either case as previously commented I'll keep this in mind for the follow up date normalization cleanup.